example/thirdparty/commons-io/build.sc

```diff
@@ -0,0 +1,77 @@
+import mill._, javalib._, publish._
+import $ivy.`com.lihaoyi::mill-contrib-jmh:$MILL_VERSION`
+import contrib.jmh.JmhModule
+
+object commonsio extends RootModule with PublishModule with MavenModule {
+  def publishVersion = "2.17.0-SNAPSHOT"
+
+  def pomSettings = PomSettings(
+    description = artifactName(),
+    organization = "org.apache.commons",
+    url = "https://github.com/apache/commons-io",
+    licenses = Seq(License.`Apache-2.0`),
+    versionControl = VersionControl.github(owner = "apache", repo = "commons-io"),
+    developers = Nil
+  )
+
+//  def ivyDeps = Agg(
+//    ivy"com.google.auto.service:auto-service:1.0",
+//  )
+//
+//  def javacOptions = Seq("-processor", "com.google.auto.service.processor.AutoServiceProcessor")
+
```

Comment:
Why are these lines commented out?

This way, they end up in the rendered documentation, outside of the code block, which is definitely not what we want.

Maybe, we can avoid it by using some spaces before the `//`.

```suggestion
  //  def ivyDeps = Agg(
  //    ivy"com.google.auto.service:auto-service:1.0",
  //  )
  //
  //  def javacOptions = Seq("-processor", "com.google.auto.service.processor.AutoServiceProcessor")

```

Reply:
dead code, will remove